Flexbox/styles.css

```diff
@@ -9,30 +10,64 @@ body {
     text-align: center;
     border: 1px solid #14213D;
     border-radius: 15px 15px 0 0;
-    height: 100px;
     margin: 10px;
     padding: 10px;
     background-color: #14213D;
     color: #FFFFFF;
 }
 
-.form-group>div {
+.form-group {
+    display: flex;
+    flex-direction: column;
     margin: 10px;
     padding: 10px;
     color: #FFFFFF;
     background-color: #14213D;
     line-height: 1.6;
 }
 
+.input-1 {
+    margin: 5px;
+    padding: 5px;
+}
+
+.input-2 {
+    margin: 5px;
+    padding: 5px;
+}
+
 .input-3 {
-    border: 0px;
+    margin: 5px;
+    padding: 5px;
+}
+
+.form-group-bottom {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #FFFFFF;
+    background-color: #14213D;
+    margin: 10px;
+    padding: 10px;
     border-radius: 0 0 15px 15px;
 }
 
 .submit-button {
-    
+    color:#FFFFFF;
+    transition-duration: 0.4s;
+    background: none;
+    border: 1px solid #FCA311;
+    border-radius: 15px;
     padding: 10px;
+}
+
+.submit-button:hover {
+    color:#FFFFFF;
+    letter-spacing: 3px;
+    cursor: pointer;
+    font-weight: bold;
     background: #FCA311;
-    border: 0px;
+    border: 1px solid #FCA311;
     border-radius: 15px;
+    padding: 10px;
```

Comment:
`color`, `border`, `border-radius` i `padding` się nie zmieniają na hover, więc nie musisz ich tu już powtarzać :) `cursor: pointer` też powinien być po prostu na stanie wyjskowym. Poza tym jest supcio, ale taki jeszcze bonus informacyjny: animacje "kosztują", mniej lub więcej w zależności od animowanej właściwości,  tzn. jeżeli np. zmieniamy letter-spacing, to będzie to mniej wydajne ("droższe") niż zmiana czegoś transformem, bo "Animating properties that change the geometry of the page (layout) or cause painting are particularly expensive." https://developers.google.com/web/fundamentals/design-and-ux/animations/animations-and-performance